src/main.rs

```diff
@@ -1,14 +1,11 @@
+use std::f32;
```

Comment:
Is this needed for something?

Reply:
Nope, not in this file. Weird that there is no unused lint. I'll remove.

Reply:
Done